lib/linter-registry.js

```diff
@@ -1,12 +1,17 @@
 /* @flow */
+/* eslint-disable import/no-duplicates */
 
+import ConfigFile from 'sb-config-file'
 import { Emitter, CompositeDisposable } from 'atom'
 import type { TextEditor, Disposable } from 'atom'
-import { linter as validateLinter, messages as validateMessages, messagesLegacy as validateMessagesLegacy } from './validate'
-import { getEditorCursorScopes, normalizeMessages, normalizeMessagesLegacy, isPathIgnored, shouldTriggerLinter, $version, $activated, $requestLatest, $requestLastReceived } from './helpers'
+
+import * as Helpers from './helpers'
+import * as Validate from './validate'
+import { $version, $activated, $requestLatest, $requestLastReceived, LINTER_CONFIG_FILE_PATH, LINTER_CONFIG_FILE_DEFAULT } from './helpers'
```

Comment:
If you put this above this line, it documents _exactly_ why you need to disable the rule:

```raw
// eslint-disable-next-line import/no-duplicates
```